bin/smash

```diff
@@ -159,7 +160,7 @@ def main():
        assert(p.returncode is not None)
 
        if p.returncode != 0:
-          if args.keep_going:
+          if args.stop_going:
```

Comment:
No, stop_going by default stores True, what means will serve as --keep-going being set.  If you set --stop-going, it will stop execution on first failed scenario, since its action is store_false and "if args.stop_going:" statement won't be executed.